src/botservice/azext_bot/.vscode/tags

```diff
@@ -0,0 +1,586 @@
+operation_entity_paged.py	../botservice\models\operation_entity_paged.py	1;"	kind:file	line:1
+operation_result_operations.py	../botservice\operations\operation_result_operations.py	1;"	kind:file	line:1
+operations.py	../botservice\operations\operations.py	1;"	kind:file	line:1
+os	../custom.py	/^import os$/;"	kind:namespace	line:7
+pdb	../custom.py	/^    import pdb$/;"	kind:namespace	line:176
+prompt_y_n	../custom.py	/^from knack.prompting import prompt_y_n  # pylint: disable=unused-import$/;"	kind:namespace	line:10
+provisionConvergedApp	../custom.py	/^def provisionConvergedApp(bot_name):$/;"	kind:function	line:22
+publish_app	../custom.py	/^def publish_app(cmd, client, resource_group_name, resource_name, git_url=None, git_token=None, git_branch='master',  # pylint: disable=inconsistent-return-statements$/;"	kind:function	line:273
+pylint	../__init__.py	/^from azext_bot._help import helps  # pylint: disable=unused-import$/;"	kind:namespace	line:7
+pylint	../_exception_handler.py	/^    from msrest.exceptions import ClientRequestError  # pylint: disable=import-error$/;"	kind:namespace	line:12
+pylint	../custom.py	/^from knack.prompting import prompt_y_n  # pylint: disable=unused-import$/;"	kind:namespace	line:10
+pylint	../custom.py	/^import adal  # pylint: disable=import-error$/;"	kind:namespace	line:9
+random	../custom.py	/^        import random$/;"	kind:namespace	line:228
+re	../custom.py	/^        import re$/;"	kind:namespace	line:226
+requests	../custom.py	/^    import requests$/;"	kind:namespace	line:331
+requests	../custom.py	/^    import requests$/;"	kind:namespace	line:48
+resource.py	../botservice\models\resource.py	1;"	kind:file	line:1
+s1	../botservice\models\azure_bot_service_enums.py	/^    s1 = "S1"$/;"	kind:variable	line:18
+sdk	../botservice\models\azure_bot_service_enums.py	/^    sdk = "sdk"$/;"	kind:variable	line:29
+service_provider.py	../botservice\models\service_provider.py	1;"	kind:file	line:1
+service_provider_parameter.py	../botservice\models\service_provider_parameter.py	1;"	kind:file	line:1
+service_provider_properties.py	../botservice\models\service_provider_properties.py	1;"	kind:file	line:1
+service_provider_response_list.py	../botservice\models\service_provider_response_list.py	1;"	kind:file	line:1
+shutil	../custom.py	/^import shutil$/;"	kind:namespace	line:8
+sku	../custom.py	/^from azext_bot.botservice.models import Bot, BotProperties, sku, BotChannel$/;"	kind:namespace	line:13
+sku.py	../botservice\models\sku.py	1;"	kind:file	line:1
+skype_channel	../botservice\models\azure_bot_service_enums.py	/^    skype_channel = "SkypeChannel"$/;"	kind:variable	line:43
+skype_channel.py	../botservice\models\skype_channel.py	1;"	kind:file	line:1
+skype_channel_properties.py	../botservice\models\skype_channel_properties.py	1;"	kind:file	line:1
+skype_create	../custom.py	/^def skype_create(client, resource_group_name, resource_name, is_disabled=None, enable_messa
```

Comment:
what is this file for?

Reply:
vs code inserts this for some reason. i will remove it

Reply:
aah generated by vscode for some reason. i will remove it. it always regenerates it every time i open the code base in vscode.